Sources/HAP/Server/Device.swift

```diff
@@ -317,6 +317,22 @@ public class Device {
         return pairingState == .paired
     }
 
+    // Remove all the pairings made with this Device
+    // Can be used in the event of a stale configuration file
+    public func unpairFromAllControllers() {
+        logger.debug("Before unpair")
```

Comment:
Do we need all this logging including the before and after state? And if so: can we condense these six lines into a single line?

Reply:
The logging is not needed at all. I can will remove them.